runtime/include/jvmti.h.m4

```diff
@@ -20,6 +20,7 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
 changequote(`[',`]')dnl
+changecom()dnl
```

Comment:
The default quotes are \` and ' - this breaks the use of `[` and `]`.

Reply:
changecom() does not affect quotes.

Reply:
Note: `changecom()dnl` is required to generate `#define JVMTI_VERSION (0x30000000 + 16 * 0x10000)` instead of `#define JVMTI_VERSION (0x30000000 + JAVA_SPEC_VERSION * 0x10000)` which won't compile in OpenJDK JCL natives such as `jdk.jdwp.agent/share/native/libjdwp/debugInit.c where JAVA_SPEC_VERSION` is not defined.
The default comment is `#`.

Reply:
Sorry, I misread it.

Rather than using `changecom`, I suggest we just quote the `#`:
```
dnl /* JVMTI_VERSION = 0x30000000 + majorversion * 0x10000 + minorversion (always 0) * 0x100 */
ifelse(eval(JAVA_SPEC_VERSION >= 15), 1, [[#]define JVMTI_VERSION (0x30000000 + (JAVA_SPEC_VERSION * 0x10000))], [
```
Also, note the added `dnl`; I don't think that comment belongs in the output.

Reply:
Updated the PR with quotation of `#`.